src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -2021,7 +2021,7 @@ protected override void BeginProcessing()
             {
                 return;
             }
-
+            
```

Comment:
Please remove extra spaces. 
Our default editor config is to trim trailing spaces - why do you disable this?

Reply:
Not sure why it was disabled for mine, but I re-enabled it.